docs/html-jsx.html

```diff
@@ -0,0 +1,12 @@
+<!DOCTYPE html>
+<html>
+<head>
+  <title>HTML to JSX</title>
+  <!-- Github Pages lacks basic functionality such as permanent server-side
+       redirects, so the best we can do is a meta redirect -->
+  <meta http-equiv="refresh" content="0; url=http://magic.reactjs.net/htmltojsx.htm" />
```

Comment:
I dream of a world where Github Pages supports server-side redirects... Maybe one day. Missing functionality is the main reason I'm not really using Github Pages for anything much.